bin/cdn-upload

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+process.on('unhandledRejection', err => console.error(err))
+
+const fs = require('fs')
+const path = require('path')
+const semver = require('semver')
+const S3 = require('aws-sdk/clients/s3')
+const Cloudfront = require('aws-sdk/clients/cloudfront')
+
+// INPUT PARAMETERS
+const packageDir = process.cwd()
+const packageJson = require(packageDir + "/package.json")
+const packageName = packageJson.name
+const packageVersion = packageJson.version
+const bucketName = 'bugsnagcdn'
+const distributionId = 'E205JDPNKONLN7'
+const callerReference = `${packageName}@${packageVersion}`
+const files = process.argv.slice(2)
+
+const s3Client = new S3()
+const cloudFrontClient = new Cloudfront()
+
+
+const upload = async (localPath, remotePath) => {
+  const relativePath = path.relative('..', localPath)
+  console.log(`uploading ${relativePath} -> ${remotePath}`)
+  
+  const fileStream = await fs.createReadStream(localPath)
+  const response = await s3Client.putObject({ 
+    Bucket: bucketName, 
+    Key: remotePath, 
+    Body: fileStream,
+    ACL: 'public-read',
+    CacheControl: 'public, max-age=315360000'
+    // TODO: Is it possible to add the Access-Control-Allow-Origin header?
+    //   'Access-Control-Allow-Origin': '*',
```

Comment:
This header was previously set on the old upload script along with the acl and cache control, however as we're using cloudfront I'm not convinced this is required

Reply:
Yeah, there's no public access to this bucket so I _think_ it's unnecessary